IntelliTect.TestTools.Selenate.Tests/DriverHandlerTests.cs

```diff
@@ -85,7 +69,7 @@ public void SetTimeoutChangesDefaultValue()
                 TimeSpan.FromSeconds(1),
                 handler
                     .GetType()
-                    .GetProperty("Timeout", BindingFlags.Instance | BindingFlags.NonPublic)
+                    .GetProperty("Timeout", BindingFlags.Instance | BindingFlags.NonPublic)!
```

Comment:
I would use null propagation here so you get an assert failure rather than a null reference exception
```suggestion
                    .GetProperty("Timeout", BindingFlags.Instance | BindingFlags.NonPublic)?
```

Reply:
Done for all instances using this.